README.md

```diff
@@ -33,6 +33,17 @@ It will not:
 * Cookbooks:
   * Sander van Zoest sysctl `https://github.com/svanzoest-cookbooks/sysctl`
 
+### Platform
+
+- Debian 6, 7, 8
+- Ubuntu 12.04, 14.04, 16.04
```

Comment:
Can we maybe remove the ubuntu 12.04 and centos 5? They are EOL

Reply:
Fedora 24 and debian 6 are EOL too

Reply:
Sure, I update and remove them from metadata, too

Reply:
We can update the metadata in the next major version, but I am not mentioning the old system in the readme.

Reply:
I would suggest to remove them from metadata too: I’m not sure if cookbook will work properly on this old distros at all. Its in my eyes only a leftover, which I missed to remove in the PRs.

Reply:
I am going to do that in another PR if you're okay with that.